snap/local/runtime-helpers/bin/edgex-cli-wrapper.sh

```diff
@@ -1,10 +1,3 @@
 #!/bin/sh -e
 
-# configuration.toml is required for the client to run
-if [ ! -f "$HOME/.edgex-cli/configuration.toml" ]; then
-    mkdir -p "$HOME/.edgex-cli"
-    cp "$SNAP/res/sample-configuration.toml" "$HOME/.edgex-cli/configuration.toml"
-    echo "Created $HOME/.edgex-cli/configuration.toml from sample configuration file."
-fi
-
 exec "$@"
```

Comment:
Is this wrapper still needed? Seem like it no longer adds any value.

Reply:
Not really, but it might be needed for the new commands to be added. If not, then I suggest removing it in the next PR.